spigot/src/main/java/de/sean/blockprot/bukkit/config/DefaultConfig.java

```diff
@@ -404,4 +407,8 @@ public boolean isLockableShulkerBox(Material type) {
     public boolean isLockableInventory(InventoryType type) {
         return lockableInventories.contains(type);
     }
+
+    public boolean isSimpleHopperProtection() {
+        return simpleHopperProtection;
+    }
```

Comment:
Do you perhaps have any information on the performance of `config.getX` functions, given that I usually just call them instead of caching the variable?

Reply:
Config gets stored in memory when the load is called, so from there everything is memory only anyways - moving over to having the bool in a dedicated variable might be a bit of an improvement but certainly not by much